cylc/flow/profiler.py

```diff
@@ -54,8 +55,5 @@ def log_memory(self, message):
         """Print a message to standard out with the current memory usage."""
         if not self.enabled:
             return
-        proc = Popen(
-            ["ps", "h", "-orss", str(os.getpid())],
-            stdin=DEVNULL, stdout=PIPE)
-        memory = int(proc.communicate()[0])
+        memory = psutil.Process(os.getpid()).memory_info().rss / 1000
```

Comment:
```suggestion
        memory = psutil.Process(os.getpid()).memory_info().rss / 1024
```
At the risk of sounding like a physics teacher - please can you write the units somewhere in the output too? It's usually guessable from context, but...

Reply:
Changed from KiB to KB. Because memory is allocated in 4KB blocks this is the most sensible base unit for profiling.

Reply:
Ugh, sources can't agree whether block size is 4KB or 4KiB.

Reply:
The fact that I have to go and look up the details of units every time suggests that the labelling is fundamentally flawed. Units shouldn't be this hard. :(